package.json

```diff
@@ -23,12 +23,12 @@
     "angular-material": "1.1.0",
     "angular-moment": "0.8.3",
     "angular-route": "1.5.8",
+    "angular-sanitize": "1.5.8",
     "angular-translate": "2.7.2",
-    "angular-ui-bootstrap": "0.14.3",
+    "angular-ui-bootstrap": "^2.1.3",
```

Comment:
nice job! Could you fix the angular-ui-bootstrap version? We don't want unexpected updates ^^ 
Thanks